ExampleReduction.ipynb

```diff
@@ -216,6 +216,23 @@
     "instrument = dbs.get_instruments_at_site('lsc', settings.db_address)[0]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Get the instrument record\n",
+    "------------------------------"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "instrument = dbs.get_instruments_at_site('lsc', settings.db_address)[0]"
+   ]
+  },
```

Comment:
lets do a 'git checkout dev banzai_nres/ExampleReduction.ipynb' to clear these changes

Reply:
Done.